src/cmd/cli/command/commands_test.go

```diff
@@ -138,26 +138,27 @@ func TestVersion(t *testing.T) {
 }
 
 func TestCommandGates(t *testing.T) {
-	mockService := &mockFabricService{canIUseResponse: defangv1.CanIUseResponse{}}
+	mockService := &mockFabricService{}
 	_, handler := defangv1connect.NewFabricControllerHandler(mockService)
 
 	server := httptest.NewServer(handler)
 	t.Cleanup(server.Close)
 
-	type cmdPermTest struct {
+	t.Setenv("AWS_REGION", "us-test-2")
+
+	if err := os.Chdir("../../../testdata/empty"); err != nil {
+		t.Fatalf("Failed to change directory: %v", err)
+	}
```

Comment:
Do we need to chdir back after the test is run?

Reply:
I wondered too, but it doesn't seem to be needed, so I assumed `testing` pkg takes care of it.